src/api/uploads/MultiputUpload.js

```diff
@@ -0,0 +1,367 @@
+/**
+ * @flow
+ * @file Multiput upload
+ * @author Box
+ */
+
+import Base from '../Base';
+import { getFileLastModifiedAsISONoMSIfPossible } from '../../util/uploads';
+import createWorker from '../../util/uploadsSHA1Worker';
+import MultiputPart, { PART_STATE_UPLOADED, PART_STATE_DIGEST_READY } from './MultiputPart';
+import type { StringAnyMap } from '../../flowTypes';
+
+const DEFAULT_MULTIPUT_CONFIG = {
```

Comment:
If we're exposing this publicly, let's bring over the documentation for all the params as well.

Reply:
done